tests/serverpod_test_server/migrations/migration_registry.txt

```diff
@@ -6,3 +6,4 @@
 
 20240125141253049
 20240725082303667
+20240816094748409
```

Comment:
Suggestion: Combine these changes with the previous migration.

We try to only introduce a single migration with every major update.
There is no need to record minor fixes since we constantly "nuke" the test docker container.

Instead of creating a new migration, please remove the last one and recreate it with your new model definition.

Reply:
Makes sense, will fix! 👍